packages/client/test/cli/cli.spec.ts

```diff
@@ -24,25 +51,145 @@ describe('[CLI]', () => {
           assert.fail(`client encountered error: ${message}`)
         }
       })
-
       child.stderr.on('data', (data) => {
         const message: string = data.toString()
         assert.fail(`stderr: ${message}`)
       })
-
       child.on('close', (code) => {
         if (typeof code === 'number' && code > 0) {
           assert.fail(`child process exited with code ${code}`)
         }
       })
     })
-  }, 10000)
-
+  }, 18000)
+  it('should start engine rpc and provide endpoint with auth disabled', async () => {
+    const file = require.resolve('../../dist/bin/cli.js')
+    const cliArgs = ['--rpcEngine', '--rpcEngineAuth=false', '--dev=poa']
+    const child = spawn(process.execPath, [file, ...cliArgs])
+    return new Promise((resolve) => {
+      child.stdout.on('data', async (data) => {
+        const message: string = data.toString()
+        if (message.includes('http://')) {
+          // if http endpoint startup message detected, call http endpoint with RPC method
+          assert.ok(message.includes('engine'), 'read from HTTP RPC')
+          assert.ok(message.includes('rpcEngineAuth=false'), 'read from HTTP RPC')
```

Comment:
Could we adjust this test to actually call the engine endpoint and confirm you can read from it rather than just reading the log messages?  Here again, the test success message should be a bit more precise, indicating that the engine RPC endpoints were started.

Reply:
fixed